sdk/security_keyvault/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "azure_security_keyvault"
-version = "0.12.0"
+version = "0.12.1"
```

For the foreseeable future, we prefer to increment the crate versions all at once to reduce difficulties with incompatibility issues for different `azure_core`. 

```suggestion
version = "0.12.0"
```